### PR TITLE
[release-4.9] OCPBUGS-1635: Update registry library dependency to pick up proxy support

### PR DESCRIPTION
**Pull request by kim-tsao**
Manual cherry-pick of #12040

### CODE REVIEW

File: vendor/OWNERS [hunk 1]
```diff
@@ -1,8 +0,0 @@
-reviewers:
```
Comment (spadgett) [inline vendor/OWNERS:1]: Did you mean to delete this?

Comment (kim-tsao) [reply vendor/OWNERS:1]: No, that's a mistake

**Comment by kim-tsao** [comment]
Devfile tracking issue: https://github.com/devfile/api/issues/934

**Comment by kim-tsao** [comment]
Updated with a fix from the registry library that's compatible with 4.9

**Comment by invincibleJai** [comment]
/retitle [release-4.10] OCPBUGS-1635: Update registry library dependency to pick up proxy support

**Comment by invincibleJai** [comment]
/retitle [release-4.9] OCPBUGS-1635: Update registry library dependency to pick up proxy support

**Comment by sanketpathak** [comment]
Verified on cluster using cluster-bot
[Screencast from 09-22-2022 020043AM.webm](https://user-images.githubusercontent.com/10252230/193601745-b3f98cca-a1aa-4ff2-9172-7c7e820398d3.webm)

/label cherry-pick-approved

**Pull request merged**